css/index.scss

```diff
@@ -244,6 +244,7 @@
 @import './components/dashboards/list';
 @import './components/dashboards/template-selector';
 @import './components/dashboards/dashboards-subheader-block';
+@import './components/dashboards/dashboards-gallery';
```

Comment:
This is probably not necessary if we reuse the `dashboard-thumbnail-list` styles file.

Reply:
Fixed here: https://github.com/resource-watch/resource-watch/pull/1185/commits/906f1bce4b882a0b0bfb84a641261c884020b757